vuepress/docs/next/docs/curate/bundle-component-details.md

```diff
@@ -0,0 +1,436 @@
+---
+sidebarDepth: 2
+---
+
+# Bundle Component Descriptors 
+
+
+Elements such as pages, content, simple widgets, fragments, and static resources are classified as platform components on Entando. This page describes each of these elements with an example.
+
+Currently, Entando supports the following component types: 
+
+![component-types.png](./img/component-types.png)
+
+Each component is defined by a descriptor YAML located in the bundle `platform` directory, inside the corresponding type as listed below. During the bundle pack phase, the descriptors are added to build the Docker images for the bundle. 
+
+Here is an example structure of a bundle project:
+
+```
+bundle-project/
+...
+  platform/
+    assets/
+    categories/
+    contentTemplates/
+    contentTypes/
+    contents/
+    fragments/
+    groups/
+    labels/
+    pageTemplates/
+      template.yaml
```

Comment:
Minor, since section is for the structure, we could probably remove the .yaml examples, or alternatively make them more specific, e.g. my-page-template.yaml, my-page.yaml, etc.